Make sure we can compile when video mode switching has been disabled.

http://bugs.gentoo.org/show_bug.cgi?id=87716

By Wormo <@gentoo.org>

--- src/draw.c
+++ src/draw.c
@@ -3847,6 +3847,7 @@
 }
 #endif
 
+#ifdef USE_XF86VM
 // Selects the mode with the highest vertical refresh frequency
 // from all modes with resolution iResX and iResY.
 // Returns -1 if no mode is found.
@@ -3871,6 +3872,7 @@
     }
     return bestmode;
 }
+#endif
 
 // Create display
 
